### DISCUSSION TITLE
Initial Version?

### DISCUSSION THREAD

**Discussion by bennypowers** [changesets/changesets#819] [Q&A]
I often find myself initializing changesets in a single-package-or-monorepo project, where:
1. We've never published before
2. We want to start in prerelease
3. We want to eventually graduate from prerelease to 1.0

Should I set the initial version in package.json to 0.0.0 or to 1.0.0?

**Comment by Andarist** [comment]
That's a good question - it's probably better to start with `0.0.0`. Just remember that if you have some packages, you don't have any changesets and you are using our GitHub action then you might immediately trigger a release after you push things out. You can read more about this "problem" here: https://github.com/changesets/changesets/pull/762

**Reply by bennypowers** [reply]
thanks i should be fine since the PR which adds `@changesets/cli` also adds everything else, including the initial chageset
